CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -844,7 +858,8 @@ public let DECL_NODES: [Node] = [
       Child(
         name: "attributes",
         kind: .collection(kind: .attributeList, collectionElementName: "Attribute", defaultsToEmpty: true),
-        nameForDiagnostics: "attributes"
+        nameForDiagnostics: "attributes",
+        documentation: "Attributes that are attached to the extension declaration."
```

Comment:
This just got me wondering: What attributes can be applied to extensions. One thing I could think of was `@MainActor`. What do you think about mentioning that as an example? I generally find that  examples help visualize what the nodes are for.

```suggestion
        documentation: """
          Attributes that are attached to the extension declaration.
          
          ### Example
          
          `@MainActor` in 
          
          ```swift
          @MainActor extension SomeType {}
          ```
          """
```